<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabButtons</class>
 <widget class="QWidget" name="TabButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnUp">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>142</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDown">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>142</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLeft">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>142</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRight">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>140</y>
     <width>142</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioADB">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>408</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ADB</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioNetwork">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>444</y>
     <width>76</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Network</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioUART">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>480</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>UART</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupMethod">
   <property name="geometry">
    <rect>
     <x>619</x>
     <y>378</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Methods :</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>151</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnPower">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Power</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnMode">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Mode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRadio">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>FM/AM</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnVideo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Video</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAudio">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Audio</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAVIn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>AVIn</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBluetooth">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Bluetooth</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSettings">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPhone">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>15</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnVolUp">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Vol. Up</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnVolDown">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Vol. Down</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnVolMute">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Vol. Mute</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>groupMethod</zorder>
  <zorder>btnUp</zorder>
  <zorder>btnDown</zorder>
  <zorder>btnLeft</zorder>
  <zorder>btnRight</zorder>
  <zorder>radioADB</zorder>
  <zorder>radioNetwork</zorder>
  <zorder>radioUART</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
 </widget>
 <tabstops>
  <tabstop>btnSettings</tabstop>
  <tabstop>btnVolUp</tabstop>
  <tabstop>btnVolDown</tabstop>
  <tabstop>btnVolMute</tabstop>
  <tabstop>btnUp</tabstop>
  <tabstop>btnDown</tabstop>
  <tabstop>btnLeft</tabstop>
  <tabstop>btnRight</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
